hbase-it/pom.xml

```diff
@@ -282,9 +287,6 @@
           <!-- TODO: failsafe does timeout, but verify does not fail the build because of the timeout.
                  I believe it is a failsafe bug, we may consider using surefire -->
           <forkedProcessTimeoutInSeconds>1800</forkedProcessTimeoutInSeconds>
-          <argLine>-enableassertions -Xmx${failsafe.Xmx}
```

Comment:
Does this really work if we move the argLine to properties?

Reply:
@Apache9  it's working. Any way will handle as @ndimiduk  mentioned so that need not move argLine.
`rajeshb+   21915   20420  0 11:13 pts/0    00:00:00 /bin/sh -c cd '/home/rajeshbabu/apache/hbase/hbase-it' && '/usr/lib/jvm/jdk-17-oracle-x64/bin/java' '-Dorg.apache.hbase.thirdparty.io.netty.tryReflectionSetAccessible=true' '--add-modules' 'jdk.unsupported' '--add-opens' 'java.base/java.io=ALL-UNNAMED' '--add-opens' 'java.base/java.nio=ALL-UNNAMED' '--add-opens' 'java.base/sun.nio.ch=ALL-UNNAMED' '--add-opens' 'java.base/java.lang=ALL-UNNAMED' '--add-opens' 'java.base/jdk.internal.ref=ALL-UNNAMED' '--add-opens' 'java.base/java.lang.reflect=ALL-UNNAMED' '--add-opens' 'java.base/java.util=ALL-UNNAMED' '--add-opens' 'java.base/java.util.concurrent=ALL-UNNAMED' '--add-exports' 'java.base/jdk.internal.misc=ALL-UNNAMED' '--add-exports' 'java.security.jgss/sun.security.krb5=ALL-UNNAMED' '--add-exports' 'java.base/sun.net.dns=ALL-UNNAMED' '--add-exports' 'java.base/sun.net.util=ALL-UNNAMED' '-enableassertions' '-Xmx4g' '-Djava.security.egd=file:/dev/./urandom' '-verbose:gc' '-XX:+PrintCommandLineFlags' '-XX:+PrintFlagsFinal' '-jar' '/home/rajeshbabu/apache/hbase/hbase-it/target/surefire/surefirebooter-20240229111348142_3.jar' '/home/rajeshbabu/apache/hbase/hbase-it/target/surefire' '2024-02-29T11-13-48_087-jvmRun1' 'surefire-20240229111348142_1tmp' 'surefire_0-20240229111348142_2tmp'
rajeshb+   21916   21915 99 11:13 pts/0    00:00:31 /usr/lib/jvm/jdk-17-oracle-x64/bin/java -Dorg.apache.hbase.thirdparty.io.netty.tryReflectionSetAccessible=true --add-modules jdk.unsupported --add-opens java.base/java.io=ALL-UNNAMED --add-opens java.base/java.nio=ALL-UNNAMED --add-opens java.base/sun.nio.ch=ALL-UNNAMED --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/jdk.internal.ref=ALL-UNNAMED --add-opens java.base/java.lang.reflect=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED --add-opens java.base/java.util.concurrent=ALL-UNNAMED --add-exports java.base/jdk.internal.misc=ALL-UNNAMED --add-exports java.security.jgss/sun.security.krb5=ALL-UNNAMED --add-exports java.base/sun.net.dns=ALL-UNNAMED --add-exports java.base/sun.net.util=ALL-UNNAMED -enableassertions -Xmx4g -Djava.security.egd=file:/dev/./urandom -verbose:gc -XX:+PrintCommandLineFlags -XX:+PrintFlagsFinal -jar /home/rajeshbabu/apache/hbase/hbase-it/target/surefire/surefirebooter-20240229111348142_3.jar /home/rajeshbabu/apache/hbase/hbase-it/target/surefire 2024-02-29T11-13-48_087-jvmRun1 surefire-20240229111348142_1tmp surefire_0-20240229111348142_2tmp
`

Reply:
Simplified the patch  as for the @ndimiduk suggestion just changed the flags according to the jvm version not moved argLine.

Reply:
> Does this really work if we move the argLine to properties?

This may require a relatively modern version of maven...